### ISSUE TITLE
Bug: Layout toggle for grid/list view is not accessible

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#3887]
The layout toggle for grid/list view is not accessible. The "buttons" are currently divs, so they are not focusable. They should probably be radio buttons.
